.gitignore

```diff
@@ -1,3 +1,4 @@
-node_modules/
-dist/
-esnext/
+node_modules/
+dist/
+esnext/
+.DS_store
```

Comment:
As per Johann's suggestion previously, we should be ignoring OS-specific files on our own machines globally in order to avoid polluting `.gitignore` with them.